openapi-filter.js

```diff
@@ -47,10 +47,23 @@ let argv = require('yargs')
     .number('maxAliasCount')
     .default('maxAliasCount',100)
     .describe('maxAliasCount','maximum YAML aliases allowed')
+    .alias('configFile', 'c')
+    .describe('configFile', 'The file & path for the filter options')
     .help()
     .version()
     .argv;
 
+// apply options from config file if present
+if (argv && argv.configFile) {
+    console.log('Options Config file used: ', argv.configFile);
+    try {
+        let configFileOptions = JSON.parse(fs.readFileSync(argv.configFile, 'utf8'));
```

Comment:
If we used the `yaml` parser we could allow people to write their config files in either JSON or YAML (as YAML is a superset of JSON).

Reply:
I was thinking the same thing about the YAML & JSON support.

I started with the JSON to get early feedback, so I'll add the YAML support to this PR

Reply:
JSON & YAML are supported in the updated PR